example/examples/GetSetClear.js

```diff
@@ -8,76 +8,62 @@
  * @flow
  */
 
-import React, {Component} from 'react';
+import React from 'react';
 import {StyleSheet, Text, View, Button} from 'react-native';
 
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
 type Props = {};
-type State = {
-  storedNumber: string,
-  needRestart: boolean,
-};
-export default class GetSet extends Component<Props, State> {
-  state = {
-    storedNumber: '',
-    needRestart: false,
-  };
 
-  async componentWillMount() {
-    const storedNumber = await AsyncStorage.getItem(STORAGE_KEY);
-    if (storedNumber) {
-      this.setState({
-        storedNumber,
-      });
-    }
-  }
+export default function GetSet(_: Props) {
```

Comment:
Yay, functional components 🎉 
nit: You can remove `Props` and this parameter

Reply:
Got a complaint about `componentWillMount` being deprecated so I converted it. Will remove `Props` parameter.